engines/xeen/dialogs/dialogs_char_info.cpp

```diff
@@ -81,147 +81,271 @@ void CharacterInfo::execute(int charIndex) {
+					intf.highlightChar(_buttonValue);
+					redrawFlag = true;
+				} else {
+					_vm->_mode = MODE_CHARACTER_INFO;
+				}
+				break;
 
-		case Common::KEYCODE_UP:
-		case Common::KEYCODE_KP8:
-			if (_cursorCell > 0) {
-				showCursor(false);
-				--_cursorCell;
-				showCursor(true);
-			}
-			w.update();
-			break;
+			case Common::KEYCODE_UP:
+			case Common::KEYCODE_KP8:
+				if (_cursorCell > 0) {
+					showCursor(false);
+					--_cursorCell;
+					showCursor(true);
+				}
+				w.update();
+				break;
 
-		case Common::KEYCODE_DOWN:
-		case Common::KEYCODE_KP2:
-			if (_cursorCell < 20) {
-				showCursor(false);
-				++_cursorCell;
-				showCursor(true);
-			}
-			w.update();
-			break;
+			case Common::KEYCODE_DOWN:
+			case Common::KEYCODE_KP2:
+				if (_cursorCell < 20) {
+					showCursor(false);
+					++_cursorCell;
+					showCursor(true);
+				}
+				w.update();
+				break;
 
-		case Common::KEYCODE_LEFT:
-		case Common::KEYCODE_KP4:
-			if (_cursorCell >= 5) {
-				showCursor(false);
-				_cursorCell -= 5;
-				showCursor(true);
-			}
-			w.update();
-			break;
+			case Common::KEYCODE_LEFT:
+			case Common::KEYCODE_KP4:
+				if (_cursorCell >= 5) {
+					showCursor(false);
+					_cursorCell -= 5;
+					showCursor(true);
+				}
+				w.update();
+				break;
+
+			case Common::KEYCODE_RIGHT:
+			case Common::KEYCODE_KP6:
+				if (_cursorCell <= 15) {
+					showCursor(false);
+					_cursorCell += 5;
+					showCursor(true);
+				}
+				w.update();
+				break;
 
-		case Common::KEYCODE_RIGHT:
-		case Common::KEYCODE_KP6:
-			if (_cursorCell <= 15) {
+			case 1001:
+			case 1002:
+			case 1003:
+			case 1004:
+			case 1005:
+			case 1006:
+			case 1007:
+			case 1008:
+			case 1009:
+			case 1010:
+			case 1011:
+			case 1012:
+			case 1013:
+			case 1014:
+			case 1015:
+			case 1016:
+			case 1017:
+			case 1018:
+			case 1019:
+			case 1020:
 				showCursor(false);
-				_cursorCell += 5;
+				_cursorCell = _buttonValue - 1001;
 				showCursor(true);
+				w.update();
+				// fall through
+
+			case Common::KEYCODE_RETURN:
+			case Common::KEYCODE_KP_ENTER: {
+				bool result = expandStat(_cursorCell, *c);
+				_vm->_mode = MODE_COMBAT;
+				if (result)
+					redrawFlag = true;
+				break;
 			}
-			w.update();
-			break;
-
-		case 1001:
-		case 1002:
-		case 1003:
-		case 1004:
-		case 1005:
-		case 1006:
-		case 1007:
-		case 1008:
-		case 1009:
-		case 1010:
-		case 1011:
-		case 1012:
-		case 1013:
-		case 1014:
-		case 1015:
-		case 1016:
-		case 1017:
-		case 1018:
-		case 1019:
-		case 1020:
-			showCursor(false);
-			_cursorCell = _buttonValue - 1001;
-			showCursor(true);
-			w.update();
-			// fall through
-
-		case Common::KEYCODE_RETURN:
-		case Common::KEYCODE_KP_ENTER: {
-			bool result = expandStat(_cursorCell, *c);
-			_vm->_mode = MODE_COMBAT;
-			if (result)
-				redrawFlag = true;
-			break;
-		}
 
-		case Common::KEYCODE_c:
-			if (Common::RU_RUS != Common::parseLanguage(ConfMan.get("language"))) {
+			// � ({�}���)
```

Comment:
Minor nitpick.. It's preferred to keep any comments in English, even if the code involved is dealing with a foreign language. If there's many such comments, it would better to change them to English.

Reply:
Most of the comments are reflecting in UTF8 what is written in the strings, for easier reference. But this particular one is a clear overlook: the file is not in utf8 but in windonws-1251, and this comment says: "В ({В}ещи", Inventory

Reply:
Fixed in later commits